### PR TITLE
Add AWS instructions for rkt on CoreOS

### PR DESCRIPTION
**Pull request by bakins**
AWS cluster scripts can also bring up a cluster using CoreOS for workers using rkt.

### CODE REVIEW

File: docs/getting-started-guides/rkt/README.md [hunk 1]
```diff
@@ -55,5 +55,34 @@ $ kube-up.sh
...
+```shell
+$ export COREOS_CHANNEL=stable
+```
+
+Then you can launch the cluster by:
+````shell
+$ kube-up.sh
+```
+
+Note that we are still working on making all containerized the master
```
Comment (rjnagal) [inline docs/getting-started-guides/rkt/README.md]: s/containerized the master/the containerized master/

File: docs/getting-started-guides/rkt/README.md [hunk 2]
```diff
@@ -55,5 +55,34 @@ $ kube-up.sh
...
+components run smoothly in rkt. Before that we are not able to run the
+master node with rkt yet.  The master node is currently always Ubuntu.
```
Comment (rjnagal) [inline docs/getting-started-guides/rkt/README.md]: drop "yet"

**Comment by rjnagal** [comment]
Thanks @bakins 
just minor typos. Also please squash the commits.

File: docs/getting-started-guides/rkt/README.md (hunk 2 above)
Comment (bakins) [inline docs/getting-started-guides/rkt/README.md]: "yet" is in the comment for GCE. Just leave that one be?

File: docs/getting-started-guides/rkt/README.md (hunk 1 above)
Comment (bakins) [inline docs/getting-started-guides/rkt/README.md]: Just a copy from the GCE above, but I can change this one.

**Comment by bakins** [comment]
Reworded master comment and squashed.

**Comment by rjnagal** [comment]
LGTM, thanks @bakins 

doc-only change, merging.

**Comment by yifan-gu** [comment]
Thanks @bakins!

**Pull request merged**